### PR TITLE
Moving to GH actions

### LINKED ISSUE #38: Move to GitHub actions
Travis is transitioning to a pay-only model and its free version is increasingly slow to use. Moving to GH actions, as in Mikado, would solve the issue.

### PR DESCRIPTION
**Pull request by lucventurini**
Closes #38 by enabling GH actions. 

Further, it moves the `etc` and `dependencies` folder to be subfolders of minos, for easier programmatic retrieval post-installation (using `pkg_resources`).

### CODE REVIEW

File: .github/workflows/minos.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        mamba-version: "*"
+        use-mamba: true
+        channels: bioconda, conda-forge, defaults
+        use-only-tar-bz2: true # IMPORTANT: This needs to be set for caching to work properly!
+        auto-update-conda: true
+        activate-environment: "test"
+        auto-activate-base: false
+    - name: Install dependencies
+      run: |
+        python setup.py install
```
Comment (cschu) [inline .github/workflows/minos.yml:47]: I thought you'd move this to bdist_wheel? Maybe I misunderstood..

**Review by cschu — COMMENTED** [review]
Nothing further (except for that one question) to comment really, as I'm not familiar with github actions.

File: .github/workflows/minos.yml (hunk 1 above)
Comment (lucventurini) [reply .github/workflows/minos.yml:47]: I made `install` and `bdist_wheel` equivalent. But good point, I will update the test

Comment (cschu) [reply .github/workflows/minos.yml:47]: I mean, whichever "works best". In the long run that should be a conda installation.

Comment (lucventurini) [reply .github/workflows/minos.yml]: Fair. In the meantime, I think that wheel will do :)

**Pull request merged**